bears/js/JSHintBear.py

```diff
@@ -16,19 +31,377 @@ class JSHintBear(LocalBear, Lint):
+                                                Browserify.
+        :param environment_devel:               This option defines globals that
+                                                are usually used for debugging:
+                                                ``console``, ``alert``, etc.
+        :param environment_dojo:                This option defines globals
+                                                exposed by the Dojo Toolkit.
+        :param environment_typed:               This option defines globals for
+                                                typed array constructors.
+        :param environment_phantom:             This option defines globals
+                                                available when your core is
+                                                running inside of the PhantomJS
+                                                runtime environment.
+        :param maxerr:                          This options allows you to set
+                                                the maximum amount of warnings
+                                                JSHintBear will produce before
+                                                giving up. Default is 50.
+        :param maxstatements:                   Maximum number of statements
+                                                allowed per function.
+        :param maxdepth:                        This option lets you control how
+                                                nested do you want your blocks
+                                                to be.
+        :param maxparams:                       Maximum number of formal
+                                                parameters allowed per function
+        :param maxcomplexity:                   Maximum cyclomatic complexity in
+                                                the code.
+        :param shadow:                          This option suppresses warnings
+                                                about variable shadowing i.e.
+                                                declaring a variable that had
+                                                been already declared somewhere
+                                                in the outer scope.
+                                                 - "inner" - check for variables
+                                                             defined in the same
+                                                             scope only
+                                                 - "outer" - check for variables
+                                                             defined in outer
+                                                             scopes as well
+                                                 - false   - same as inner
+                                                 - true    - allow variable
+                                                             shadowing
```

Comment:
If the bools are python values, please write them uppercase -> `False` and `True` since they exist like this in python :)